### PR TITLE
Fix code format (include indentation and remove dead code)

### PR DESCRIPTION
**Pull request by fengshunli**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: apis/fluentd/v1alpha1/plugins/params/model.go [hunk 1]
```diff
@@ -73,12 +74,12 @@ func (ps *PluginStore) Hash() string {
 	return utils.HashCode(c.String())
 }
 
-// Returns tag value
+// GetTag the Returns tag value
```
Comment (wenchajun) [inline apis/fluentd/v1alpha1/plugins/params/model.go:77]: `GetTag  Returns tag value`

Comment (fengshunli) [reply apis/fluentd/v1alpha1/plugins/params/model.go]: done

**Comment by fengshunli** [comment]
done @benjaminhuo

**Comment by benjaminhuo** [comment]
> done @benjaminhuo

Still the new comments are incorrect

**Pull request closed without merging**